install/kubernetes/helm/istio/values-istio-minimal.yaml

```diff
@@ -43,4 +43,7 @@ global:
   
   useMCP: false
 
+  mtls:
+    auto: false
```

Comment:
Should this be true?

Reply:
minimal one does not even have citadel installed.